src/Admin/AbstractAdmin.php

```diff
@@ -2573,9 +2574,7 @@ protected function buildForm(): void
     /**
      * Gets the subclass corresponding to the given name.
      *
-     * @param string $name The name of the sub class
-     *
-     * @return string the subclass
+     * @phpstan-return class-string
```

Comment:
e.g. this method has more specific type hints here: https://github.com/sonata-project/SonataAdminBundle/pull/6361/files#diff-61e6d1f8fafe87b868adbe28ce0ba8f6R3474

There are also some other methods that have different type hints.

Reply:
Fixed